doc/manual/manual.tex

```diff
@@ -9681,6 +9681,69 @@ \subsubsection{Thin shell gravity benchmark}
+Inside the shell, $\rho=\rho_0$, yielding
+\begin{equation}
+g=\frac{\partial U}{\partial r} = \frac{4 \pi}{3} G \rho_0 r + \frac{A}{r^2},
+\end{equation}
+where $A$ is another integration constant. 
+At the inner boundary, $r=R\textsubscript{inner}$ and $g=0$, allowing $A$ to be computed. Substituting in the value of $A$,
+\begin{equation}
+g=\frac{\partial U}{\partial r} = \frac{4 \pi}{3} G \rho_0 
+\left(r - \frac{R\textsubscript{inner}^3}{r^2} \right). \label{eqgin}
+\end{equation}
+When $r\geq R\textsubscript{outer}$, the gravitational potential is given by Eq. (\ref{eq:app1}). Requiring the gravity field to be continuous at $r=R\textsubscript{outer}$:
+\begin{equation}
+g(r) = \frac{G M}{r^2} \label{eqgout},
+\end{equation}
+where $M=\frac{4 \pi}{3} \rho_0(R\textsubscript{outer}^3-R\textsubscript{inner}^3)$ is the mass contained in the shell.
+For $r\ge R\textsubscript{outer}$, the potential is obtained by integrating Eq.(\ref{eqgout}):
+\begin{equation}
+U(r)=-\frac{GM}{r} +D,
+\end{equation}
+where $D$ is an integration constant which has to be zero since we require the potential to vanish for $r\rightarrow \infty$.
+The potential within the shell, $R\textsubscript{inner}\leq r \leq R\textsubscript{outer}$, is found by integrating Eq.~\eqref{eqgin}:
+\begin{equation}
+U(r)= \frac{4 \pi}{3} G \rho_0 \left(\frac{r^2}{2} + \frac{R\textsubscript{inner}^3}{r} \right)  + E,
+\end{equation}
+where $E$ is a constant. Continuity of the potential at $r=R\textsubscript{outer}$ requires that $E=-2\pi\rho_0 G R\textsubscript{outer}^2$. 
+Gravitational acceleration is zero for $r\leq R\textsubscript{inner}$, so the potential there is constant and a continuity requirement yields
+\begin{equation}
+U(r)=2\pi G \rho_0 (R\textsubscript{inner}^2-R\textsubscript{outer}^2).
+\end{equation}
+
+The gravity postprocessor in \aspect{} can be used to calculate the radial components of gravity ($g_r$ and $U$) at arbitrary points using the Sampling scheme `\textit{list of points}'. For this benchmark we calculate points along a line from the center of the planet to a distant point, $r=0$ to $r=10,000~\si{\km}$ (Figure \ref{fig:grav-thick-shell}). Arbitrarily, the latitude and longitude are both set to $13\si{\degree}$ so as to avoid potential measurement artefacts due to symmetry. The list of radii is defined as follows:
+
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/gravity_thick_shell/doc/thick_shell.prm.out}
+
+The resulting measurements obtained for a mesh composed of 12 caps of $32^3$ cells (i.e., 393,216 total mesh cells) are shown in Fig.~(\ref{fig:grav-thick-shell}) and are in good agreement with the analytical profiles.
```

Comment:
Is 12 correct? How do you get a coarse mesh for that?

Reply:
393216/12=32768=12*32^3. Also it checks out visually.

Reply:
I had thought that we start with either 6 or 24 or 96 cells. But I had to look it up and you're right that the choices are 6, 12 or 96. The default is 96, so what you really seem to have is 16^3 * 96?

Reply:
![Screenshot from 2021-07-13 21-45-30](https://user-images.githubusercontent.com/7646114/125515439-f28e9612-9e77-48aa-a1ac-3dcb0b101758.png)

then it is confusing because this is the mesh and it looks like the citcom mesh which is made of 12 surface 'diamonds' of 16x16 elements.

Reply:
Ah, nice investigative work :-) I must be wrong!